res/controllers/Numark_DJ2GO2_Touch.midi.xml

```diff
@@ -221,6 +221,78 @@
           <script-binding/>
         </options>
       </control>
+      <control>
```

Comment:
DJ2GO2/DJ2GO2 Touch does not support "shift" button, but send another control (control + 8) when push "Pad" holding "PAD MODE" in "CUES" mode.  
Other DJ software assigns it to clear hotcue.  
In other modes, holding "PAD MODE" is ignored. DJ2GO2/DJ2GO2 Touch sends same control holding "PAD MODE" or not.